### DISCUSSION TITLE
Is Uno.Toolkit free for large enterprise companies?

### DISCUSSION THREAD

**Discussion by vitorelli** [unoplatform/uno.toolkit.ui#1003] [Q&A] [answered]
On Uno toolkit page https://platform.uno/uno-toolkit/ states:

_Uno Toolkit is available for free for individual developers and businesses with revenue of less than USD 1,000,000._

Does Uno.Toolkit.UI nuget library require a license ? Are all components free?

https://www.nuget.org/packages/Uno.Toolkit.UI/

**Comment by sasakrsmanovic** [comment, marked as answer]
Hello @vitorelli . Currently it is not and it requires a license to be granted over email, in exchange for some rudimentary market/product research questions answered. Please reach out to me at sasha at platform.uno to get a license.
